build/lib/Crypto_bot_pkg/crypto_selection.py

```diff
@@ -0,0 +1,17 @@
+class crypto_bot:
+    """
+    A class where you can select your favorite cryptos, track them and use a trading bot    
+    """
+
+    def which_crypto():
+        """
+        This is function 1
```

Comment:
once you've made the function, don't forget to add parameters to the documentation

Reply:
Yes I'd second that!